### PR TITLE
#6 Spring Boot upgrade, use spring-boot-web to expose health endpoint

### PR DESCRIPTION
**Pull request by thibauult**
Actually, quite a lot of stuffs in this PR: 
- upgraded BDK and Spring Boot versions
- replaced `com.sun.net.httpserver.HttpServer` (used for HealthCheck) by a Spring `RestController`
- testing infinite retries of the DF Loop, to avoid it crashing when pod is unavailable 
- added Maven Wrapper

### CODE REVIEW

File: .mvn/wrapper/MavenWrapperDownloader.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/*
```
Comment (thibauult) [inline .mvn/wrapper/MavenWrapperDownloader.java:1]: This file can be ignored from review

File: .mvn/wrapper/maven-wrapper.properties [hunk 2]
```diff
@@ -0,0 +1,2 @@
+distributionUrl=https://repo.maven.apache.org/maven2/org/apache/maven/apache-maven/3.6.3/apache-maven-3.6.3-bin.zip
```
Comment (thibauult) [inline .mvn/wrapper/maven-wrapper.properties:1]: This file can be ignored from review

File: mvnw [hunk 3]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (thibauult) [inline mvnw:1]: This file can be ignored from review

File: mvnw.cmd [hunk 4]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (thibauult) [inline mvnw.cmd:1]: This file can be ignored from review

**Review by symphony-youri — APPROVED** [review]

**Pull request merged**